libs/core/langchain_core/prompts/loading.py

```diff
@@ -127,13 +127,26 @@ def _load_prompt(config: dict) -> PromptTemplate:
 
 def load_prompt(path: Union[str, Path]) -> BasePromptTemplate:
     """Unified method for loading a prompt from LangChainHub or local fs."""
+    if path.startswith("azureml://"):
+        return _load_prompt_from_azureai(path)
     if hub_result := try_load_from_hub(
         path, _load_prompt_from_file, "prompts", {"py", "json", "yaml"}
     ):
         return hub_result
     else:
         return _load_prompt_from_file(path)
 
+def _load_prompt_from_azureai(path: Union[str, Path]) -> BasePromptTemplate:
```

Comment:
i think we are trying to use `load_prompt` less, so would rather make this a top level function

eg we have `hub.pull` to pull things from hub, which we use more